Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -185,7 +185,10 @@ var TouchableNativeFeedback = React.createClass({
 
   _handleResponderMove: function(e) {
     this.touchableHandleResponderMove(e);
-    this._dispatchHotspotUpdate(e.nativeEvent.pageX, e.nativeEvent.pageY);
+    var [elementWidth, elementHeight] = (this.pressInLocation)
+    ? [this.pressInLocation.pageX - e.nativeEvent.locationX, this.pressInLocation.pageY - e.nativeEvent.locationY]
+    : [0, 0];
+    this._dispatchHotspotUpdate(e.nativeEvent.pageX - elementWidth, e.nativeEvent.pageY - elementHeight);
```

Comment:
I know the value we pass to dispatchHotspotUpdate needs to be the touch location relative to the frame of the button.

What do values in pressInLocation and e.nativeEvent represent? I feel like we could simplify the calculation here.